software/sw_build.mk

```
@@ -63,14 +64,23 @@ TEMPLATE_LDS=src/$@.lds
 
 IOB_SOC_OPENCRYPTOLINUX_CFLAGS ?=-Os -nostdlib -march=rv32imac -mabi=ilp32 --specs=nano.specs -Wcast-align=strict
 
-IOB_SOC_OPENCRYPTOLINUX_INCLUDES=-I. -Isrc 
+IOB_SOC_OPENCRYPTOLINUX_INCLUDES=-I. -Isrc -Isrc/crypto/McEliece -Isrc/crypto/McEliece/common
```

We can include some sources only if running on the FPGA:
```suggestion
IOB_SOC_OPENCRYPTOLINUX_INCLUDES=-I. -Isrc
ifneq ($(SIMULATION),1)
IOB_SOC_OPENCRYPTOLINUX_INCLUDES+=-Isrc/crypto/McEliece -Isrc/crypto/McEliece/common
endif
```

---

I did not do this because the includes in gcc is for header lookups and stuff. None of which actually adds code so it's not a problem if they get added regardless.